Tavisca.Bootcamp.LanguageBasics.Exercise2/Program.cs

```diff
@@ -13,18 +13,60 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string[] postTimes, string[] showTimes, string expected)
-        {
+        private static void Test(string[] postTimes, string[] showTimes, string expected)
+        {
             var result = GetCurrentTime(postTimes, showTimes).Equals(expected) ? "PASS" : "FAIL";
             var postTimesCsv = string.Join(", ", postTimes);
             var showTimesCsv = string.Join(", ", showTimes);
-            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
+            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
         }
 
-        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
+            int n = exactPostTime.Length;
+
+            for(int i=0;i<n;i++)  {
+                for(int j=i+1;j<n;j++)  {
+                    if(exactPostTime[i] == exactPostTime[j])  {
+                        if(showPostTime[i] != showPostTime[j])
+                            return "impossible";
+                    }
+                }
+            }
+
+            String[] result = new string[n];
+
+    	    for(int i=0;i<n;i++)  {
+                String[] timeSplit = exactPostTime[i].Split(":");
+
+                //int hour = int.Parse(timeSplit[0]);
+                //int minute = int.Parse(timeSplit[1]);
+                //int second = int.Parse(timeSplit[2]);
+
+                DateTime date = new DateTime(2019, 6, 24, int.Parse(timeSplit[0]), int.Parse(timeSplit[1]), int.Parse(timeSplit[2]), 24);
+
+                if(showPostTime[i].Contains("seconds"))  {
+                    result[i] = exactPostTime[i];

```

Comment:
Do we need to maintain an array here? Can we do this using a string variables only?